beacon-chain/blockchain/forkchoice/process_block_test.go

```diff
@@ -126,11 +125,7 @@ func TestStore_UpdateBlockAttestationVote(t *testing.T) {
 	defer testDB.TeardownDB(t, db)
 	params.UseMinimalConfig()
 
-	deposits, _, _ := testutil.SetupInitialDeposits(t, 100)
-	beaconState, err := state.GenesisBeaconState(deposits, uint64(0), &ethpb.Eth1Data{BlockHash: make([]byte, 32)})
-	if err != nil {
-		t.Fatal(err)
-	}
+	beaconState, _, _ := testutil.DeterministicGenesisState(100)
```

Comment:
This isn't right. If you aren't going to handle the error returned by these methods then you should provide `testing.T` and handle the failure within the helper method. The purpose of passing `t` was to avoid all of the `if err != nil { t.Fatal(err) }` boilerplate.